doc/Learn/Storage/HowTo-RequiredEntitlements.md

```diff
@@ -0,0 +1,64 @@
+---
+uid: Uno.Extensions.Storage.HowToRequiredEntitlements
+---
+# How-To: Adding Required Entitlements
+
+On Apple platforms (iOS, Mac Catalyst) the Uno storage extension use the OS Key Chain service to store secrets. This requires your application to have the [`keychain-access-groups`](https://developer.apple.com/documentation/bundleresources/entitlements/keychain-access-groups) entitlement to work properly.
+
+## Step-by-steps
+
+> [!IMPORTANT]
+> This guide assumes you used the template wizard or `dotnet new unoapp` to create your solution. If not, it is recommended that you follow the [instructions](xref:Uno.Extensions.HowToGettingStarted) for creating an application from the template.
+
+### 1. Adding the Entitlements.plist file
+
+The default location, inside your project, for the new file(s) should be:
+
+* iOS: `iOS/Entitlements.plist`
+* Mac Catalyst: `MacCatalyst/Entitlements.plist`
+
+The content of the file(s) should be:
+
+```xml
+<key>keychain-access-groups</key> 
+ <array> 
+ 	<string>$(AppIdentifierPrefix)$(CFBundleIdentifier)</string> 
```

Comment:
Does $(AppIdentifierPrefix) work for catalyst - I seem to recall seeing debug warnings about this not resolving when trying to get signing to work

Reply:
It's definitively needed for `keychain-access-groups` since it comes from the provisioning profile and that's how you can get sharing to work across apps (using something like `$(AppIdentifierPrefix)*` so all apps, with the same PP, can share their secrets).

> I seem to recall seeing debug warnings about this not resolving when trying to get signing to work

That does not ring a bell - unless that happens only when not signing ?